website/pages/en/cookbook/sorting.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: GraphQL Sorting Strategies
+---
+
+## Sorting Entity IDs
+
+When creating subgraphs in The Graph, it's important to consider the type of ID used for your entities. This choice can have a significant impact on the performance and ease of use of your subgraph.
+
+If accurate sorting is desired, both Strings and Bytes for IDs are not optimal as the sorting is done non-sequentially which can lead to unexpected results. If attempting to sort by id of type "ID" (which is a string), the ordered sequence might look like this:
```

Comment:
What is accurate? Only for numerical values, sorting by `Bytes` and `String` feels off. If the ID is a transactionHash or a string, that sorting is fine.

So I suggest to be clear here.